store/cache.js

```
@@ -22,12 +22,18 @@
-      `https://www.warframe.com/repos/weeklyRivens${rootGetters['worldstate/platform'].toUpperCase()}.json`
+      `https://www-static.warframe.com/repos/weeklyRivens${rootGetters['worldstate/platform'].toUpperCase()}.json`
     );
-    const raw = await res.text();
-    if (!(raw && raw.length)) return;
-    const rivens = JSON.parse(raw.replace(/NaN/g, 0).replace(/WARNING:.*\n/, ''));
-    commit('commitRivens', [rootGetters['worldstate/platform'], rivens]);
+    try {
+      const raw = await res.text();
+      if (!(raw && raw.length)) return;
+      // eslint-disable-next-line no-eval
+      const rivens = eval(raw);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)